lib/smart_proxy_salt/salt.rb

```diff
@@ -14,10 +14,19 @@ class Plugin < ::Proxy::Plugin
 
       default_settings :autosign_file      => '/etc/salt/autosign.conf',
                        :salt_command_user  => 'root',
-                       :use_api            => false
+                       :use_api            => false,
+                       :saltfile           => '/etc/foreman-proxy/settings.d/salt.saltfile'
```

Comment:
Note to self: we already write this in the same salt.yaml file and no installer change is needed.